specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/payments.json

```diff
@@ -0,0 +1,711 @@
+{
+  "consumes": [
+    "application/json"
+  ],
+  "definitions": {
+    "DetachPaymentMethodEligibilityError": {
+      "description": "The details of the error.",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "details": {
+          "description": "The list of detach payment method eligibility errors.",
+          "items": {
+            "$ref": "#/definitions/DetachPaymentMethodErrorDetails"
+          },
+          "readOnly": true,
+          "type": "array"
+        },
+        "message": {
+          "description": "Error message.",
+          "readOnly": true,
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "DetachPaymentMethodEligibilityErrorCode": {
+      "description": "Error code of the detach payment method eligibility validation response.",
+      "enum": [
+        "AzureSubscriptions",
+        "RecurringCharges",
+        "ReservedInstances",
+        "OutstandingCharges",
+        "PendingCharges"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "detachPaymentMethodEligibilityErrorCode"
+      }
+    },
+    "DetachPaymentMethodEligibilityResult": {
+      "description": "Result of the detach payment method eligibility.",
+      "properties": {
+        "errorDetails": {
+          "description": "The list of detach payment method eligibility errors.",
+          "items": {
+            "$ref": "#/definitions/DetachPaymentMethodErrorDetails"
+          },
+          "readOnly": true,
+          "type": "array"
+        },
+        "isEligible": {
+          "description": "Specifies whether the payment method is eligible to be detached from the billing profile.",
+          "readOnly": true,
+          "type": "boolean"
+        }
+      },
+      "type": "object"
+    },
+    "DetachPaymentMethodErrorDetails": {
+      "description": "Error details of the detach payment method eligibility validation.",
+      "properties": {
+        "code": {
+          "$ref": "#/definitions/DetachPaymentMethodEligibilityErrorCode",
+          "description": "Error code for the detach payment method eligibility validation."
+        },
+        "message": {
+          "description": "Error message for the detach payment method eligibility validation.",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "PaymentMethod": {
+      "allOf": [
+        {
+          "$ref": "./types.json#/definitions/ArmResource"
+        }
+      ],
+      "description": "A payment method.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PaymentMethodProperties",
+          "title": "Payment method properties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object"
+    },
+    "PaymentMethodLink": {
```

Comment:
that was a feedback but I found out that it's a conscious choice to keep 2 objects that currently are somewhat similar because paymentMethod is open for expansion but  PaymentMethodLink is only supposed to contain a subset of those properties as relevant to BG.